llvm/unittests/ADT/SmallSetTest.cpp

```diff
@@ -17,6 +17,66 @@
 
 using namespace llvm;
 
+TEST(SmallSetTest, ConstructorIteratorPair) {
+  auto L = {1, 2, 3, 4, 5};
```

Comment:
Could you make the type explicit, both here and everywhere else below. Either initializer list or some other vector type is fine.